Jetton Mint & Dedust liquidity deposit

1IxI1 opened the pull request.

1IxI1 committed:
Add jetton_mint action. Fixes, WIP

1IxI1 committed:
Dedust liquidity deposit, refactor

2 action types: deposit and deposit first asset. Small refactoring,
still quite WIP

verdigos commented:
Приведи пожалуйста по схеме бд к следующему виду
```
dex_deposit_liquidity_data = Column(CompositeType("dex_deposit_liquidity_details", [
        Column("dex", String),
        Column("amount1", Numeric),
        Column("amount2", Numeric),
        Column('asset1', String),
        Column('asset2', String),
        Column('user_jetton_wallet_1', String),
        Column('user_jetton_wallet_2', String),
        Column("lp_tokens_minted", Numeric),
    ]))
```
Нужно удалить `AndMatcher`. Вместо него в `DedustDepositBlockMatche`r использовать поле `children_matchers`. Плюс там же (193я строка) вместо `DedustDeployDepositContract.opcode` использовать `DestroyLiquidityDepositContract.opcode `
Так же вместо `ContractMatcher(opcode=None,  optional=True)` -  `BlockTypeMatcher( 'ton_transfer', optional=True)`

1IxI1 committed:
Liq.: add user jwallets

1IxI1 committed:
Change db, fixes

Fix a bug, prepare db for merge: now deposit_contract in
destination_secondary column

1IxI1 committed:
Clean comments

1IxI1 committed:
Liquid staking withdraw/deposit

WIP. doesn't recogn. TONStakersWithdrawRequest.opcode

1IxI1 committed:
Reorder db type

1IxI1 committed:
Add nominator pool deposit/withdraw request

Needs fix in indexer/indexer/events/blocks/messages/staking.py#L196
